### PR TITLE
Update changelog for 2.3.2

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 
+## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.2) (Unreleased)
+
+* Fix high CPU load due to insane internal sync clock values [#4312](https://github.com/mixxxdj/mixxx/pull/4312) [lp1943320](https://bugs.launchpad.net/mixxx/+bug/1943320)
+* Fix Auto DJ skipping random tracks [#4319](https://github.com/mixxxdj/mixxx/pull/4319) [lp1941989](https://bugs.launchpad.net/mixxx/+bug/1941989)
+* Fix resetting main window title when all decks are stopped (a regression 2.3.1). [#4296](https://github.com/mixxxdj/mixxx/pull/4296)
+* Fix preference option for re-analyzing beatgrids imported from other software [#4288](https://github.com/mixxxdj/mixxx/pull/4288)
+* Numark DJ2GO2 Touch: Fix sampler, hotcue, beatloop buttons _scripts.js [#4287](https://github.com/mixxxdj/mixxx/pull/4287)
+* Improve Hercules inpulse 300 mapping [#4246](https://github.com/mixxxdj/mixxx/pull/4246)
+* Improve Denon MC7000 [#4021](https://github.com/mixxxdj/mixxx/pull/4021) [#4324](https://github.com/mixxxdj/mixxx/pull/4324)
```
Comment (Holzhaus) [inline CHANGELOG.md:11]:
```suggestion
* Numark DJ2GO2 Touch: Fix sampler, hotcue and beatloop buttons [#4287](https://github.com/mixxxdj/mixxx/pull/4287)
* Hercules Inpulse 300: Add better FX controls and other minor improvemrnts [#4246](https://github.com/mixxxdj/mixxx/pull/4246)
* Denon MC7000: Improve slip mode and jog wheel handling [#4021](https://github.com/mixxxdj/mixxx/pull/4021) [#4324](https://github.com/mixxxdj/mixxx/pull/4324)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.2) (Unreleased)
+
+* Fix high CPU load due to insane internal sync clock values [#4312](https://github.com/mixxxdj/mixxx/pull/4312) [lp1943320](https://bugs.launchpad.net/mixxx/+bug/1943320)
+* Fix Auto DJ skipping random tracks [#4319](https://github.com/mixxxdj/mixxx/pull/4319) [lp1941989](https://bugs.launchpad.net/mixxx/+bug/1941989)
```
Comment (Holzhaus) [inline CHANGELOG.md:6]:
```suggestion
* Fix Auto DJ skipping tracks randomly [#4319](https://github.com/mixxxdj/mixxx/pull/4319) [lp1941989](https://bugs.launchpad.net/mixxx/+bug/1941989)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.2) (Unreleased)
+
+* Fix high CPU load due to insane internal sync clock values [#4312](https://github.com/mixxxdj/mixxx/pull/4312) [lp1943320](https://bugs.launchpad.net/mixxx/+bug/1943320)
```
Comment (Holzhaus) [inline CHANGELOG.md:5]:
```suggestion
* Fix high CPU load due to extremely high internal sync clock values [#4312](https://github.com/mixxxdj/mixxx/pull/4312) [lp1943320](https://bugs.launchpad.net/mixxx/+bug/1943320)
```

**Review by Holzhaus — COMMENTED** [review]
Thanks. Some comments.

**Comment by daschuer** [comment]
Done and rebased to rerun the commit hook.

**Comment by uklotzde** [comment]
This does not match #4335?!?! This chaos is concerns me.

**Comment by daschuer** [comment]
This is because #4335 has a merge commit into the 2.3.1 branch. 
The alternative would be to drop the 2.3.1 branch and release from the 2.3 branch. 
I would prefer that to avoid this cris-cross merge. But the idea wa rejected by @Be-ing

**Comment by daschuer** [comment]
I can prepare a another release PR for the 2.3 branch and close all the rest if you prefer that as well. 
This would be create at least a clean history. What do you think?

**Comment by Holzhaus** [comment]
I agree. I think a separate 2.3.1 branch was unneeded in the first place. It's still possible to tag a specific commit retroactively, it doesn't need to be the branch head.

**Comment by daschuer** [comment]
https://github.com/mixxxdj/mixxx/pull/4335 now targets the 2.3 branch. 
Please have a look. This one is obsolete.

**Pull request closed without merging**